Frameworks/AddressBook/ABAddressBookManagerInternal.h

```diff
@@ -17,10 +17,25 @@
 #pragma once
 
 #import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABRecord.h>
 
 @interface _ABAddressBookManager : NSObject
```

Comment:
now that this interface actually has functions, it may be useful to declare them here so that it's easy to see what the interface does at a glance